.github/workflows/go-mod-tidy.yml

```diff
@@ -0,0 +1,25 @@
+name: Run go mod tidy
+on:
+  schedule:
+    # Do this every week on Friday moring
+    - cron: '0 7 * * FRI'
+
+jobs:
+  pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          ref: master
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+      - run: go mod tidy
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Cool. Will this show up as a PR from you? Neat.

Reply:
https://github.com/jmhodges/howsmyssl/pull/294 is an example of what one of these looks like